packages/components/psammead-inline-link/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead-inline-link",
-  "version": "1.2.0-alpha.1",
+  "version": "1.2.1-alpha.1",
```

Comment:
Should this not be `1.2.0-alpha.2`?

Reply:
Or at least `1.2.1-alpha.0`

Reply:
would agree, but I was just trying to fit in with the previous version bumping which has been doing this style https://github.com/bbc/psammead/pull/1365/files#diff-77e5a30c00e558e1ef1b46ece522934dR8